<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInfoToClassForm</class>
 <widget class="QWidget" name="AddInfoToClassForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление информации в класс</string>
  </property>
  <widget class="QPushButton" name="add_inf_add_information_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить инфомацию в класс</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_inf_class_name_line_edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_inf_protocol_name_line_edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="add_inf_class_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Класс</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_inf_protocol_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Протокол</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_inf_inf_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Информация</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
